lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -888,6 +888,7 @@ class NGPHistoryParser {
       timeChangeEvents.push(timeChange);
     }
 
+    const lastEvent = this.events[this.events.length - 1];
```

Comment:
Just checking: is it guaranteed that the last event will always contain a timestamp?